aiida/workflows/arithmetic/multiply_add.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Implementation of the MultiplyAddWorkChain for testing and demonstration purposes."""
+from aiida.orm import Code, Int
+from aiida.engine import calcfunction, WorkChain, ToContext
+from aiida.plugins.factories import CalculationFactory
+
+ArithmeticAddCalculation = CalculationFactory('arithmetic.add')
+
+
+@calcfunction
+def multiply(x, y):  # pylint: disable=invalid-name
+    return x * y
+
+
+class MultiplyAddWorkChain(WorkChain):
+    """WorkChain for testing and demonstration purposes."""
+
+    @classmethod
+    def define(cls, spec):
+        """Specify inputs and outputs."""
+        super(MultiplyAddWorkChain, cls).define(spec)
```

Comment:
```suggestion
        super().define(spec)
```
That is Python 2 syntax, which we no longer support so we simoplify it